docs/guides/getting-started.md

```diff
@@ -0,0 +1,95 @@
+---
+page_title: Getting Started
+---
+
+# Getting Started
+
+## Installing Terraform
+
+See [this guide](https://learn.hashicorp.com/tutorials/terraform/install-cli#install-terraform) for more information.
+
+## Declaring a Task
+
+In an empty directory:
+
+1. Create a directory named `results` to save the task `script`'s results.
+2. Create a file named `main.tf` with the following contents:
+
+```hcl
+terraform {
+  required_providers {
+    iterative = {
+      source  = "iterative/iterative"
+    }
+  }
+}
+
+provider "iterative" {}
+
+resource "iterative_task" "example" {
+  name  = "example"
+  cloud = "aws" # or any of: gcp, az, k8s
+
+  directory = "${path.root}/results"
+
+  script = <<-END
+    #!/bin/bash
+    echo "Hello World!" > greeting.txt
+  END
+}
+```
+
+## Initializing Terraform
+
+```console
+$ terraform init
+```
+
+This command will:
+
+1. Download and install the Iterative Provider.
+2. Initialize Terraform in the current directory.
+
+~> **Note** None of the subsequent commands will work without first setting some [authentication environment variables](https://registry.terraform.io/providers/iterative/iterative/latest/docs#authentication)
+
+## Launching Tasks
+
+```console
+$ terraform apply
+···
+```
+
+This command will:
+
+1. Create all the required cloud resources.
+2. Upload the specified results `directory` to the cloud.
```

Comment:
That's why I don't believe “results” to be the best name. Why in the earth would users **upload** _results_ anywhere?

Reply:
@casperdcl has, in the documentation review, but I would prefer a more neutral name.

Reply:
Totally agree... a task does not need to have results per se

Reply:
I think you've changed to "shared" (rather than "results" or "directory") directory? lgtm.